src/components/main/Scenario.js

```diff
@@ -95,7 +100,7 @@ class Scenario extends React.Component {
 
                 <RandomHPPDInfo  onInfoChange={this.handleInfoChange} />
                 <StaffAdd onStaffChange={this.handleStaffChange} onStaffAdd={this.handleStaffAdd} staffs={this.state.staffs} />
-                <StaffList staffs={this.state.staffs} ></StaffList>
+                <StaffList staffs={this.state.staffs} parent={this}></StaffList>
```

Comment:
You can't just pass "this" as it is unclear you need to be more specific. Like this.(function name you are passing)